xml/security_ldap_replication.xml

```diff
@@ -0,0 +1,469 @@
+     replication delay.
+    </para>
+   </listitem>
+  </itemizedlist>
+  <para>
+   In general, as LDAP is "low-write", these factors mean that all servers are
+   at least up to a common baseline of a known consistent state. Small changes
+   occur on top of this baseline, so many of these aspects of delayed
+   replication are not perceived in day to day usage.
+  </para>
+ </sect2>
+
+ <sect2 xml:id="sec-security-ldap-replication-topology">
+  <title>Designing your topology</title>
+  <para>
+   Consider the following factors when you are designing your replication
+   topology.
+  </para>
+  <itemizedlist>
+   <listitem>
+    <para>
+     The need for replication: high availability, geo-location, read scaling,
+     or a combination of all.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     How many replicas (nodes, servers) you plan to have in your topology.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Direction of data flows, both inside of the topology, and data flowing
+     into the topology.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     How clients will balance across nodes of the topology for their requests
+     (multiple ldap URIs, SRV records, load balancers).
+    </para>
+   </listitem>
+  </itemizedlist>
+  <para>
+   These factors all affect how you may create your topology. (See
+   <xref linkend="sec-security-ldap-replication-two-replicas"/> for some
+   topology examples.)
+  </para>
+ </sect2>
+
+ <sect2 xml:id="sec-security-ldap-replication-topologies">
+  <title>Example replication topologies</title>
+  <para>
+   The following sections provide examples of replication topologies, using two
+   to six &ds389; nodes. The maximum number of supported read-write replicas in
+   a topology is twenty. Operational experience shows the optimal number for
+   replication efficiency is a maximum of eight.
+  </para>
+  <para>
+   A node of the topology can only be offline for a maximum of days up to the
+   limit of the changelog (see FOO).
+  </para>
+  <sect3 xml:id="sec-security-ldap-replication-two-replicas">
+   <title>Two replicas</title>
+   <example xml:id="ex-ldap-replication-two-replicas">
+    <title>Two replicas</title>
+<screen>
+┌────┐       ┌────┐
+│ S1 │◀─────▶│ S2 │
+└────┘       └────┘
+</screen>
+   </example>
+   <para>
+    In <xref linkend="ex-ldap-replication-two-replicas"/> there are two nodes,
+    S1 and S2, which replicate bi-directionally between each other. S1 and S2
+    could be in separate data centers, or in the same data center. Clients can
+    balance across the servers using LDAP URIs, a load balancer, or DNS SRV
+    records. This is the simplest topology for high availability. Note that
+    each server needs to be able to provide 100% of client load, in case the
+    other server is offline for any reason. A two-node replication is generally
+    not adequate for horizontal read scaling.
```

Comment:
Why not appropriate - it's because in the case one server is offline, the other is required to service 100% of read traffic. 

Also worth noting that this is the "default" topology that should be considered for most deployments until proven otherwise due to how simple it is to manage.